pkg/controller/user/user_controller.go

```diff
@@ -194,32 +196,27 @@ func (r *ReconcileUser) updateUserCR(ctx context.Context, parentInstance *regist
 	user *registriesv1alpha1.User, result reconcile.Result) (reconcile.Result, error) {
 	if originalUser == nil || user == nil {
 		return reconcile.Result{},
-			fmt.Errorf("cannot update user '%s' because the original user is nil", user.Spec.Name)
+			fmt.Errorf("cannot update user because the original user is nil")
```

Comment:
Think it's better to not mention `nil` at this point since it is language specific. How about this?
```suggestion
			fmt.Errorf("cannot update user because the original user has not been set")
```